Styling for /tutors

pwaller815 opened the pull request.

Commit:
Styling for tutor list page.

Commit:
Slight change to /tutors.

Commit:
Slight change to /tutors.

EthanCloin commented on app/static/js/calendar-util.js:
merge the main branch into your `pwaller` branch to prevent overwriting changes from my last few commits. i would expect no changes to this JS file or the tutor-availability.html file in the PR

EthanCloin commented on app/templates/layout.html:
```diff
@@ -1,17 +1,18 @@
+  <meta name="viewport" content="width=
+    , initial-scale=1.0" />
+  <link href="/static/css/main.css" rel="stylesheet" />
+  {% block head %}{% endblock %}
```
i like including a block in the head tag to add stylesheets as needed so keep that. there could be a way to improve the addition of the navbar element:

just include the html for navbar minus the `head` block. here's a snippet from GPT

```html
<!-- templates/layout.html -->
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>{% block title %}Default Title{% endblock %}</title>
    <link rel="stylesheet" href="{{ url_for('static', filename='css/base.css') }}">
    {% block stylesheets %}{% endblock %}
</head>
<body>
    {% include 'navbar.html' %}
    {% block content %}{% endblock %}
</body>
</html>
```

EthanCloin commented on app/static/css/tutors.css:
```diff
@@ -0,0 +1,61 @@
+    padding: 30px 25% 30px 25%;
+}
+
+.tutor-list-header h1 {
```
consider adding a class to the element instead of referencing by tag type. not a problem bc it works exactly the same here, just a best practice. i have been aiming to follow the [BEM standard ](https://getbem.com/naming/) in this project, but i am new to it.

Commit:
Merge branch 'main' into pwaller

Commit:
Updated to feedback.

Commit:
Updated /tutors.

Commit:
Updated tutor cards.

EthanCloin commented on app/static/js/ui-util.js:
```diff
@@ -0,0 +1,31 @@
+    window.open(url, '_blank');
+}
+
+document.addEventListener("DOMContentLoaded", function () {
```
this is a really cool solution for the readmore problem! i think it should still work when the text comes from the database too.

EthanCloin committed:
Minor Refactors:

- put svg into a template so i don't have to look at those numbers
- replace custom js fxn with anchor tag to redirect
- put navbar css into the layout base template
- use url_for fxn where possible

EthanCloin reviewed (approved):
appreciate your efforts here, looks really clean!
